azext_iot/_params.py

```diff
@@ -378,3 +381,70 @@ def load_arguments(self, _):
 
     with self.argument_context('iot dps registration list') as context:
         context.argument('enrollment_id', help='ID of enrollment group')
+
+    with self.argument_context('iot dt') as context:
+        context.argument('repo_login', options_list=['--repo-login', '--rl'],
+                         help='This command supports an entity connection string with rights to perform action. '
+                         'Use to avoid PnP endpoint and repository name if repository is private. '
+                         'If both an entity connection string and name are provided the connection string takes priority.')
+        context.argument('interface', options_list=['--interface', '-i'],
+                         help='Target interface name. This should be the name of the interface not the urn-id.')
+        context.argument('command_name', options_list=['--command-name', '--cn'],
+                         help='IoT Plug and Play interface command name.')
+        context.argument('command_payload', options_list=['--command-payload', '--cp', '--cv'],
+                         help='IoT Plug and Play interface command payload. '
+                         'Content can be directly input or extracted from a file path.')
+        context.argument('interface_payload', options_list=['--interface-payload', '--ip', '--iv'],
+                         help='IoT Plug and Play interface payload. '
+                         'Content can be directly input or extracted from a file path.')
+        context.argument('source_model', options_list=['--source', '-s'],
+                         help='Source to get model definition. ',
```

Comment:
Is it well known/well documented how to use or configure `source_model`? Err on the side of more details if it makes sense.

Reply:
updated the text